### PR TITLE
Update pocket_privacy_policy_row.md

### PR DESCRIPTION
**Pull request by erenieris**
Updating Pocket Privacy Notice for cookie compliance in ROW.

### CODE REVIEW

File: en/pocket_privacy_policy.md
Comment (peiying2) [inline en/pocket_privacy_policy.md:52]: Should be `opt in`.

File: en/pocket_privacy_policy.md
Comment (peiying2) [inline en/pocket_privacy_policy.md:53]: Should be 
- `third-party`
- `opt out`

File: en/pocket_privacy_policy.md
Comment (peiying2) [inline en/pocket_privacy_policy.md:6]: Curly quotes should be used.

File: en/pocket_privacy_policy.md
Comment (peiying2) [inline en/pocket_privacy_policy.md:116]: lower case `w` as in `website` to be consistent with the rest of the doc.

**Review by peiying2 — COMMENTED** [review]
The changes are made to the existing Pocket PN, not `row.md`. Is  this intended? Do you want to change the file name?

File: en/pocket_privacy_policy.md
Comment (flodolo) [inline en/pocket_privacy_policy.md:4]: This date needs to be changed accordingly.

File: en/pocket_privacy_policy.md
Comment (flodolo) [reply en/pocket_privacy_policy.md:6]: I can fix that in a commit before merging, otherwise reviewing the diff gets unnecessarily hard.

File: en/pocket_privacy_policy.md
Comment (flodolo) [inline en/pocket_privacy_policy.md:50]: preference -> preferences? That's how the rest of the document uses it.

**Review by flodolo — COMMENTED** [review]
This document is being translated and should be delivered today or tomorrow. 

We should not merge this PR until that translation has been delivered, otherwise things will get out of control. We should have a separate request to translate only these updates.

**Comment by flodolo** [comment]
> The changes are made to the existing Pocket PN, not `row.md`. Is this intended? Do you want to change the file name?

That's adding ROW compliance to the existing privacy notice?

File: en/pocket_privacy_policy.md
Comment (flodolo) [inline en/pocket_privacy_policy.md:1]: Just noticed this: it should probably say "Pocket Privacy Policy" instead of just "Privacy Policy"?

Comment (erenieris) [reply en/pocket_privacy_policy.md:1]: Actually on Pocket's website it just says "Privacy Policy" and since it doesn't live under the Moz privacy notices (at Moz.org) I think that's ok.

**Review by flodolo — APPROVED** [review]
This is ready for me, but we need to wait for the delivery of translations from HTP to make sure we have a usable diff to give the vendor.

**Pull request merged**
